libs/ng-utils/src/lib/directives/ng-render-in.directive.spec.ts

```diff
@@ -63,3 +67,61 @@ describe('Directive: NgRenderIn', () => {
     directive?.ngOnDestroy();
   });
 });
+
+describe('Directive: NgRenderInBrowserDirective', () => {
+  let spectator: SpectatorDirective<NgRenderInBrowserDirective>;
+  let directive: NgRenderInBrowserDirective;
+
+  const createDirective = createDirectiveFactory({
+    directive: NgRenderInBrowserDirective,
+    imports: [CommonModule],
+  });
+
+  beforeEach(() => {
+    spectator = createDirective(
+      `<div *ngRenderInBrowser="or serverTpl" #container>
+          <div #content></div>
+        </div>
+
+        <ng-template #serverTpl>
+          <div #serverContainer></div>
+        </ng-template>`
+    );
+
+    directive = spectator.directive;
+  });
+
+  it('should render serverTpl', () => {
+    const serverTpl = spectator.query('#serverTpl');
+    const browserTpl = spectator.query('container');
+
+    expect(browserTpl).toBeFalsy();
+    expect(serverTpl).toBeDefined();
+  });
+});
+
+describe('NgRenderInServerDirective', () => {
+  let spectator: SpectatorDirective<NgRenderInServerDirective>;
+  let directive: NgRenderInServerDirective;
```

Comment:
Nem tudom submittálni a reviewt kommentelt sor nélkül - de itt is a nem használt változó